workflows/dags/literature/hep_create_dag.py

```diff
@@ -1191,6 +1232,7 @@ def save_and_complete_workflow(**context):
         >> get_approved_match_task
         >> check_is_update_task
         >> [merge_articles(get_approved_match_task), update_inspire_categories]
+        >> is_record_relevant()
```

Comment:
most probably this will be used as a branch  so as to trigger the correct task afterwards, but for now it is ok . We can also leave a note for this